packages/api-files/src/plugins/models/filesSettings.model.ts

```diff
@@ -34,6 +36,8 @@ export default ({ createBase }) => {
                 value: {},
                 instanceOf: withFields({
                     installed: boolean({ value: false }),
+                    uploadMinFileSize: number({ value: 0 }),
```

Comment:
Please apply validation on `uploadMinFileSize` and `uploadMaxFileSize` fields.

```
uploadMinFileSize: number({ value: 0, validation.create("required,gte:0") }),
uploadMaxFileSize: number({ value: 0, validation.create("required") }),
```

Reply:
For `uploadMaxFileSize`, I didn't put a limit. Not sure what would be a good one really. I guess we can leave it for now as is.